tasks/move-granules/tests/test-index.js

```diff
@@ -177,9 +174,15 @@ test.serial('should add input files to returned granule event.moveStagedFiles is
 
 test.serial('Should move renamed files in staging area to final location.', async (t) => {
   const newPayload = buildPayload(t);
-  const renamedFile = `s3://${t.context.stagingBucket}/file-staging/MOD11A1.A2017200.h19v04.006.2017201090724.hdf.v20180926T131408705`;
-  newPayload.input.push(renamedFile);
-  await uploadFiles(newPayload.input, t.context.stagingBucket);
+  const renamedFile = `s3://${t.context.stagingBucket}/file-staging/subdir/MOD11A1.A2017200.h19v04.006.2017201090724.hdf.v20180926T131408705`;
+  t.context.filesToUpload.push(renamedFile);
```

Comment:
Perhaps this doesn't matter, but would updating `t.context.filesToUpload` potentially affect other tests?

Reply:
I see that it was doing something similar before, but in that case the affected payload was scoped only to this test, I think?

Reply:
In the version of `ava` we use (`0.25`), `t.context` is unique to the text it's in. I suppose it wouldn't be bad idea to clone it, though, in case we ever update our version of ava.

Reply:
8b62ab6a0